lib/EventListener.php

```diff
@@ -222,18 +222,44 @@ protected function getUsersForShares(array $shares, string $owner): array {
 			}
 		}
 
+		$groupAddedUsers = false;
 		if (!empty($groups)) {
 			foreach ($groups as $gid) {
 				$group = $this->groupManager->get($gid);
 				if ($group instanceof IGroup) {
 					foreach ($group->getUsers() as $user) {
+						$groupAddedUsers = true;
 						$users[] = $user->getUID();
 					}
 				}
 			}
 		}
 
-		return array_unique($users);
+		$users = array_unique($users);
+
+		if (!$groupAddedUsers) {
+			return $users;
+		}
```

Comment:
I need to check how circle shares work. :eyes:

Reply:
I did not know that calendars can be shared with circles...

Reply:
Raised https://github.com/nextcloud/event_update_notification/issues/107